packages/api/src/controllers/stream.ts

```diff
@@ -700,7 +708,8 @@ app.get("/:id", authorizer({}), async (req, res) => {
   let stream = await db.stream.get(req.params.id);
   if (
     !stream ||
-    ((stream.userId !== req.user.id || stream.deleted) && !req.user.admin)
+    ((stream.userId !== req.user.id || stream.deleted) && !req.user.admin) ||
+    stream.projectId !== req.project?.id
```

Comment:
I think the `admin` checks should override the project Id checks, just like the override the userId check today. This applies to several other places in the PR. This would look like
```ts
((stream.userId !== req.user.id || stream.projectId !== req.project?.id || stream.deleted) && !req.user.admin)
```

Reply:
Fixed